tftb/generators/analytic_signals.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from numpy import pi
 from scipy.signal import hilbert
-from frequency_modulated import fmconst
+from .frequency_modulated import fmconst
```

Comment:
Ideally we shouldn't use relative imports. Even what I wrote was wrong, this should be:
```python
from tftb.generators.frequency_modulated import fmconst
```